bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_proc_card.dat

```diff
@@ -0,0 +1,34 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.6.1                 2017-12-12         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+define p = g u c d s u~ c~ d~ s~ b b~ 
+define j = g u c d s u~ c~ d~ s~ b b~ 
+#define l+ = e+ mu+
+#define l- = e- mu-
+
+import model sm
+import model ALP
+
+#generate p p > h, (h > ALP ALP, ( ALP > mu+ mu- ) )
+generate p p > h > Z alp , Z > l+ l- , alp > a a
```

Comment:
Please remove the uncommented+unused lines.
And here l+ l- is used while you uncommented it.
Won't you consider tau > leptonic decays at all?
Also please check whether your model card default setting is 5f or 4f.

Reply:
Removed other lines. Added tau to be consistent with DYJets.